### PR TITLE
build(deps): upgrade node version

### PR DESCRIPTION
**Pull request by michawai**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,31 +1,20 @@
 version: 2
 
 jobs:
-  node10:
+  previous_node:
     docker:
-      - image: circleci/node:10.23 # Current BAS node version
+      - image: circleci/node:14 # Future BAS node version
```
Comment (allacherkes) [inline .circleci/config.yml:6]: previous

Comment (allacherkes) [inline .circleci/config.yml:6]: you dont use?

Comment (michawai) [reply .circleci/config.yml:6]: No need to run on node 10 and 14. Not relevant anymore.

Comment (michawai) [reply .circleci/config.yml:6]: ?

Comment (allacherkes) [reply .circleci/config.yml:6]: fix comments Future->Previous

**Review by allacherkes — APPROVED** [review]

**Pull request merged**